docs/1.0. Certificate Provisioning.md

```diff
@@ -147,32 +149,33 @@ The EST Server MUST present a valid TLS Server Certificate, this Certificate MUS
 
 ### EST Server - Client Authentication
 
-The EST Server MUST authenticate the EST Client that is requesting a TLS Certificate manually or automatically using a TLS Client Certificate.
+The EST Server MUST authenticate the EST Client that is requesting a TLS (Server?) Certificate manually or automatically using a TLS Client Certificate.
```

Comment:
Lower down https://github.com/AMWA-TV/nmos-certificate-provisioning/pull/18/files#diff-1b2fb902da0028870248b51d70e5fb08a0e2281864d88f4184b15e58e666627aR172

It MUST be server and client

```suggestion
The EST Server MUST authenticate the EST Client that is requesting a TLS Certificate manually or automatically using a TLS Client Certificate.
```

Reply:
To clarify, @prince-chrismc, is it two certificates being requested, not one?
Also, I struggled to parse "manually or automatically using a TLS Client Certificate". That needs a comma or two at least to make it clear.

Reply:
It's one certificate with two purposes.

I found that unclear as well, what would manually be?

Reply:
```suggestion
The EST Server MUST authenticate the EST Client before issuing a TLS Certificate as described below.
```